docs/site/src/demos/app-bar/app-bar.md

```diff
@@ -0,0 +1,11 @@
+# App Bar
+
+The [app-bar](https://material.io/guidelines/layout/structure.html#structure-app-bar), formerly known as the action bar in Android, is a special kind of toolbar that’s used for branding, navigation, search, and actions.
+
+## Simple App Bar
+
+{{demo='demos/app-bar/SimpleAppBar.js'}}
+
+## Buttons App Bar
```

Comment:
'## Button App Bar' ? (no 's') (or 'App Bar with buttons'?)

Regarding the capitalisation, we don't seem to be consistent here across demos. I'm thinking perhaps we follow the same casing everywhere, including the title, as Google do. The exception there seems to be Floating Action Button, but that may just be a mistake by whichever designer wrote that page, as other than the page title, it's uncased on that page.